src/processCollectedSignatures.js

```diff
@@ -68,6 +73,10 @@ async function processCollSignatures(signatures, foreignChainId){
     let nonce = await getNonce(web3Foreign, VALIDATOR_ADDRESS);
     nonce = Math.max(dbNonce.foreign, nonce);
     await asyncForEach(signatures, async (colSignature, indexSig) => {
+      if(requiredBlockConfirmations > 1) {
```

Comment:
What is the reason for this check? Even if `requiredBlockConfirmations` is 1, it's useful to call the function because that wait it will wait until the transaction is mined.

Reply:
The event belongs to a transaction that was already mined so I added that check to avoid calling `waitForBlockConfirmations` if only 1 block was needed